docs/java-developer-docs

```diff
@@ -0,0 +1,171 @@
+# Learnings and Feedbacks from Java Spring-Boot SDK Usage
+
+Helium-Java is a Spring-Boot framework Java based reference application for using the Azure Web App for Containers service.
+Walmart is a big Java shop and has > 80% Spring framework usage hence Spring-Boot was chosen for building this reference app.
+
+## Following issues were uncovered and CSE feedbacks Filed as part of Helium-Java journey so far:
```

Comment:
From a longer term (post TLT) developer documentation perspective, we should remove references to CSE/CSE Feedback and links to internal sites since this is public.